### ISSUE TITLE
Spam prevention: blacklist messages

### ISSUE THREAD

**Issue by carlbennett** [carlbennett/BNRBot#10]
Create a feature that allows the user to blacklist certain messages, possibly using regular expressions or exact matching.

**Comment by carlbennett** [comment]
Snippet of text to work with:

```
[2:24:36 AM] <]{LiK`EcSTACY`2 -: [ gBot v3.0 ] [ Linux 3.5.4-hardened-r1 ] [ Rob ] :->
[2:24:41 AM] <Charlie -- ViLe Chat -- By Charlie & Mike -->
[2:24:49 AM] <{SF}WoLVeR!ne> SuPeR Friends - DC v1.3b2 [Private Edition].
[2:25:05 AM] <StaLKa> Another happy user - DC v1.3b2 [Private Edition].
[2:25:18 AM] <wizard -- Get some -- HoLyBoT v3.0b1 by HoLyOnE. -->
[2:25:34 AM] <]i$[DooFiO]i$[ is a Death Chat v0.8 by Unknown [Shinigami Edition].>
[2:25:41 AM] <l2k-Shadow l2uthless Chat Extremely Super Secret Creator Version - By: l2k-Shadow>
[2:26:06 AM] <]{LiK`Theruler` -- poop -- HoLyBoT v3.0b1 by HoLyOnE. -->
[2:26:18 AM] <]{LiK`FLeX` CurseBot v2 [Priv9 Hacker Edition]>
[2:26:47 AM] <{EwR}MeaL> hmm? - DC v1.3b2 [Private Edition].
[2:27:41 AM] <l2k-Shadow l2uthless Chat Extremely Super Secret Creator Version - By: l2k-Shadow - Idle for 2 minutes.>
[2:29:01 AM] <TeK> ][)aRkKiNgZ - DC v1.3b2 [Private Edition].
[2:29:27 AM] <Mr.Bischoff We are in control.. - DC v1.3b2 [Private Edition].>
[2:29:34 AM] <Mike> Uptime CPU: 7d, 14h, 36m, 47s - Bot: 6h, 42m, 6s - Con: 2h, 34m, 6s - Asgard3 (Dev)B: 3756 By Mike
[2:29:37 AM] <{][)K}Fluxion blah - DC v1.3b2 [Private Edition].>
```

**Comment by carlbennett** [comment]
Resolved in 02e56b5a0350f5e53d69d23512b2b87bde26e59b.
